x/crosschain/types/cctx.go

```diff
@@ -146,6 +146,10 @@ func (m *CrossChainTx) AddRevertOutbound(gasLimit uint64) error {
 		},
 		TssPubkey: m.GetCurrentOutboundParam().TssPubkey,
 	}
+
+	if m.InboundParams != nil {
+		revertTxParams.CoinType = m.InboundParams.CoinType
+	}
```

Comment:
This change is out of scope for this pr but might be small enough that it can be included here 
Ideally coin-type should be higher, as inbound, outbound[0], and outbound [1] should all have the same cointype .

Reply:
Can we add a comment mentioning it will be refactoring in the future to use a cctx wide value